docs/extend/modular-resources/create/_index.md

```diff
@@ -63,24 +63,26 @@ The Go code for the custom model (<file>mybase.go</file>) and module entry point
 {{< tabs name="Sample SDK Code">}}
 {{% tab name="Python"%}}
 
-<file>my_base.py</file> implements a custom model of the base component built-in resource, "mybase".
+<file>my_base_client.py</file> implements a custom model of the base component built-in resource, "mybase".
```

Comment:
I find this line a bit confusing, also my_base_client would be an inaccurate name - my_base_model/my_base would be better.
```suggestion
<file>my_base.py</file> implements "mybase", a custom model of the base component.
```

Reply:
That's good with me, thank you for the suggestion!